kymatio/numpy.py

```diff
@@ -1,4 +1,6 @@
-from .scattering1d.frontend.numpy_frontend import ScatteringNumPy1D as Scattering1D
+from .scattering1d.frontend.numpy_frontend import (
+    ScatteringNumPy1D as Scattering1D,
+    TimeFrequencyScatteringNumPy1D as TimeFrequencyScattering1D)
```

Comment:
why is `kymatio.numpy.TimeFrequencyScattering1D` exposed, but `kymatio.torch.TimeFrequencyScattering1D` is not?

Reply:
`jtfs-min` is [numpy only](https://user-images.githubusercontent.com/16495490/139905940-29c47d6c-4ca5-478b-bf35-c19f6f4c2867.png)